_posts/2020-12-02-spack-user-survey-2020.md

```diff
@@ -0,0 +1,918 @@
+the scripts we used to generate these charts
+[here](https://github.com/spack/spack-user-survey). There are a lot of
+results -- let us know if you find anything interesting that we missed.
+
+## Thanks!
+
+Thanks to the 169 users who filled out the survey, both for this feedback
+and for your continuing contributions to Spack! This project wouldn't be
+possible without the community!
+
+# Demographics
+
+In this section of the survey, we tried to understand the composition of
+the Spack community.
+
+## ECP and Spack
+
+The first question we asked was whether respondents were part of ECP.
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/pie_in_ecp.svg">
+    <img src="/assets/images/spack-user-survey-2020/pie_in_ecp.svg">
+  </a>
+</figure>
+
+ECP is just over a third of the community (~35%, or 61 of 169
+respondents). It includes people from both sides of the U.S. Department
+of Energy -- [NNSA](https://www.energy.gov/nnsa/) laboratories like
+[LLNL](https://llnl.gov), [LANL](https://lanl.gov), and
+[Sandia](https://sandia.gov); and
+[Office of Science](https://www.energy.gov/science/), laboratories like
+[ANL](https://www.anl.gov), [ORNL](https://ornl.gov),
+[LBL](https://www.lbl.gov/), [PNNL](https://www.bnl.gov/),
+[BNL](https://www.bnl.gov/), and others. Under ECP, the Spack team is
+focusing on delivering a software stack for the first U.S. exascale
+machines, which includes upcoming systems like:
+
+* LBL's [Perlmutter](https://www.nersc.gov/systems/perlmutter/): AMD CPU /
+  NVIDIA GPU (pre-exascale)
+* ANL's [Aurora](https://www.alcf.anl.gov/aurora): Intel CPU / Intel GPU
+* ORNL's [Frontier](https://www.olcf.ornl.gov/frontier/): AMD CPU / AMD GPU
+* LLNL's [El Capitan](https://www.llnl.gov/news/llnl-and-hpe-partner-amd-el-capitan-projected-worlds-fastest-supercomputer): AMD CPU / AMD GPU
+
+All of these are HPE/Cray systems, but the hardware is quite diverse
+(particularly the GPUs). We wanted to see whether the users at the
+bleeding edge of HPC had significantly different needs from the Spack
+community as a whole. So, in most of the following sections, we present
+responses for Spack and for ECP separately.
+
+## What kind of user are you?
+
+We asked users what their role was at their organization.
+
+<figure>
+  <a href="/assets/images/spack-user-survey-2020/two_pies_user_type.svg">
+    <img src="/assets/images/spack-user-survey-2020/two_pies_user_type.svg">
+  </a>
+</figure>
+
+Spack was originally targeted at user support teams and system
+administrators, but by far the largest parts of the community are end
+users (scientists/researchers) installing software on HPC machines (41%)
+and software developers (35%). System administrators were ~11% of the
+overall community, and user support staff were only 8%. In ECP, this is
+even more pronounced -- only a small fraction of respondents identified
+as system administrators, and developers were nearly 44% of the ECP user
```

Comment:
Looking at the blue & orange segments of the pies, are the percentages backward?
```suggestion
users (scientists/researchers) installing software on HPC machines (35%)
and software developers (41%). System administrators were ~11% of the
overall community, and user support staff were only 8%. In ECP, this is
even more pronounced -- only a small fraction of respondents identified
as system administrators, and developers were nearly 43% of the ECP user
```